docker-compose.yml

```diff
@@ -0,0 +1,65 @@
+version: "3.9"
+
+services:
+  app:
+    build:
+      args:
+        user: opendor-dev
+        uid: 1000
+      context: ./
+      dockerfile: .docker/php/Dockerfile
+    image: php:8.0
+    restart: unless-stopped
+    working_dir: /var/www/
+    volumes:
+      - ./:/var/www
+    depends_on:
+      - db
+    networks:
+      - opendor
+
+  db:
+    image: postgres:12.7
+    restart: unless-stopped
+    environment:
+      POSTGRES_DB: ${DB_DATABASE}
+      POSTGRES_USER: ${DB_USERNAME}
+      POSTGRES_PASSWORD: ${DB_PASSWORD}
+    volumes:
+      - psql:/var/lib/postgresql/data
+    ports:
+      - "5432:5432"
+    networks:
+      - opendor
+
+  nginx:
+    build:
+      context: ./
+      dockerfile: .docker/nginx/Dockerfile
+    restart: unless-stopped
+    ports:
+      - 8000:80
+      - 80:80
+    volumes:
+      - ./:/var/www
+    depends_on:
+      - app
+    networks:
+      - opendor
+
+  redis:
+    image: "redis:alpine"
+    ports:
+      - "${REDIS_PORT:-6379}:6379"
```

Comment:
Okay, love that env setting even more - how does it work with the Laravel `.env`?
Could you apply this configurable thing to all host ports?

Reply:
Only DB port needs update.. changing it.

Reply:
And which env file will be used? Is there a special `.env.docker` or something?

Reply:
Laravel `.env` will be used.. no extra configuration is needed, Just configure `.env` however you want, It will use those port username passwords etc etc etc.. 

Isn't it cool ? 😎